### DISCUSSION TITLE
Profiles

### DISCUSSION THREAD

**Discussion by warpigsmash** [EveOPlus/eve-o-preview#13] [Ideas]
It would be incredibly useful to be able to have multiple config profiles, and the ability to have 3 groups. As a corollary, it would  be useful to be able to assign an entire group to a pair of up/down hotkeys, rather than having individual hotkeys per character.
Say, for example, I have 9 accounts,, each with 1 toon. Each toon does different tasks in different scenarios.
It would be beneficial to be able to have a PVP config, a Mining config, a PVE config, a BLOPs config, etc., without having to manually rework the script each time, or move scripts in and out of the folder.
I'm hoping that this isn't a difficult feature to add.

**Comment by EveOPlus** [comment]
This has been delivered in the latest updates.
